packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_flavor_config/__brick__/android/environments/sit/google-services.json

```diff
@@ -7,7 +7,7 @@
   "client": [
     {
       "client_info": {
-        "mobilesdk_app_id": "Make sure you add your own Firebase SIT config.",
+        "mobilesdk_app_id": "Make sure you add your own Firebase development config.",
```

Comment:
Same as above

Reply:
<img width="224" alt="image" src="https://github.com/Prime-Holding/rx_bloc/assets/32831935/2cec52de-3db5-4e08-83a1-fdf0dab2595e">
Please remove the not relevant Run configuration -> main.dart

Reply:
It seems you've been using an older version of the code. This has been fixed in one of the previous commits. Please  check again and let me know if it still occurs. Also, make sure you delete the previously generated project.